### DISCUSSION TITLE
How To Drag-Drop On Differnt TreeView?

### DISCUSSION THREAD

**Discussion by 18862237583** [dotnetcore/BootstrapBlazor#2456] [Q&A]
Now I Want To  Drag-Drop On Differnt TreeView,But  I Can't Use BootstrapBlazor drag-drop Component To Reach My Goal.
Please Help Me!
Thanks Very Much

**Comment by ArgoZhang** [comment]
@18862237583 drag item from TreeA to TreeB?
